api/src/test/java/org/apache/iceberg/util/RandomUtil.java

```diff
@@ -29,6 +29,10 @@ public class RandomUtil {
 
   private RandomUtil() {}
 
+  public static String generateString(int length, Random random) {
```

Comment:
Why not just make `randomString` public?  Seems like we're just duplicating here.

Reply:
I think I was just trying to match the other public method names for this, and to make few changes to how strings are generated internally. Then I guess it ended up with the duplicate signature from the separate changes. I'll make the internal method call this one.